bitnami/kubeapps/values.yaml

```diff
@@ -1403,271 +1403,6 @@ kubeops:
     automountServiceAccountToken: true
     annotations: {}
 
-## @section Assetsvc parameters
-
-## Assetsvc parameters
-## @deprecated
-assetsvc:
```

Comment:
The main change in this PR is the removal of the `asset-svc` component (hence the major version bump) . However, this component has been disabled by default for a while.